ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -117,7 +117,17 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     # Install a nnstreamer library and examples
     ninja -C build install
 
+    # Clone nnstreamer examples repository
```

Comment:
It is my trivial proposal. :)
* before: # Clone nnstreamer examples repository
* after: # Clone the nnstreamer-example repository

Reply:
I updated this.